ENA_GBIF_Mapper/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.11
+LABEL authors="mernst"
+
+VOLUME /app
+
+WORKDIR /app
+
+COPY . /app
+
+RUN pip install -r requirements.txt
+
+EXPOSE 80
+
+CMD ["python", "run.py"]
```

Comment:
We often switch to a specific user so the application doesn't run as root in the container.
Can be done by adding:
```
# Create new user with UID
RUN adduser --disabled-password --gecos '' --system --uid 1001 python && chown -R python /app

# Set user to newly created user
USER 1001
```